packages/block-editor/src/components/global-styles/background-panel.js

```diff
@@ -492,9 +492,9 @@ function BackgroundSizeToolsPanelItem( {
 					label={ __( 'Fixed' ) }
 				/>
 			</ToggleGroupControl>
-			{ sizeValue !== undefined &&
-			sizeValue !== 'cover' &&
-			sizeValue !== 'contain' ? (
+			{ currentValueForToggle !== undefined &&
```

Comment:
Testing if `currentValueForToggle` is a good substitute for sizeValue to determine if the size control should be displayed

Reply:
That's working nicely for me!